test/client-checkout-integration-test.js

```diff
@@ -229,6 +242,44 @@ suite('client-checkout-integration-test', () => {
     });
   });
 
+  test('it resolves with a checkout on Client.checkout#updateShippingAddress', () => {
+    const checkoutId = checkoutShippingAddressUpdateV2Fixture.data.checkoutShippingAddressUpdateV2.checkout.id;
+
+    fetchMock.postOnce(apiUrl, checkoutShippingAddressUpdateV2Fixture);
+
+    return client.checkout.updateShippingAddress(checkoutId, shippingAddress).then((checkout) => {
+      assert.equal(checkout.id, checkoutId);
+      assert.equal(checkout.shippingAddress.name, 'Bob Norman');
+      assert.equal(checkout.shippingAddress.provinceCode, 'KY');
+      assert.equal(checkout.shippingAddress.countryCode, 'US');
+      assert.ok(fetchMock.done());
+    });
+  });
+
+  test('it resolves with userErrors on Client.checkout#updateShippingAddress with invalid address', () => {
+    const checkoutShippingAdddressUpdateV2WithUserErrorsFixture = {
```

Comment:
More of a nitpick - it would be good to move this fixture into the fixture file. An example can be found in https://github.com/Shopify/js-buy-sdk/blob/master/fixtures/paginated-images-fixtures.js